docs/site/content/docs/latest/packages-create-example.md

```diff
@@ -176,7 +176,7 @@ Before a Tanzu package can be deployed in a cluster, it must be made discoverabl
 
    Following the naming convention for kapp-controller YAMLs, the custom resource is saved in file `v1.0.0.yaml` (below) to indicate that it is for a specific version. Note that the name attribute references the same qualified name in the previous Metadata CR.
 
-   ```sh
+    ```sh
     apiVersion: data.packaging.carvel.dev/v1alpha1
```

Comment:
The rest of the block should probably be dedented too, right?

Reply:
The opening of the code block is only indented 3 spaces, the rest of it is 4 so I just indent the opening one more space which aligns the rest of it correctly.

Reply:
Oh, sorry, you're absolutely right. :)